frontend_v2/src/app/components/home/home.component.ts

```diff
@@ -50,6 +50,7 @@ export class HomeComponent implements OnInit, AfterViewInit, OnDestroy {
    */
   ngAfterViewInit() {
     this.SUBSCRIBE_FORM[this.subscribeForm] = this.components;
+    (<any>window).twttr.widgets.load();
```

Comment:
@gautamjajoo did you test what happens when `load()` call fails? If yes, how?

Reply:
Actually we don't need `load()` without it the code would work

Reply:
@gautamjajoo please remove it

Reply:
Done